index.html

```diff
@@ -31,21 +31,21 @@
       <div class="row">
         <div class="col-md-4 col-md-offset-4">
           <ul class="list-inline social-buttons">
-              <li>
-                  <a target="_blank" href="https://www.facebook.com/gutechsoc"><i class="fab fa-facebook-f"></i></a>
-              </li>
-              <li>
-                  <a target="_blank" href="https://twitter.com/gutechsoc"><i class="fab fa-twitter"></i></a>
-              </li>
-              <li>
-                <a target="_blank" href="https://www.youtube.com/channel/UCbwnhXjLXAWxc6Nj_z-tlyQ/videos?view=57&flow=grid"><i class="fab fa-youtube"></i></a>
-              </li>
-              <li>
-                  <a target="_blank" href="https://discord.gg/EGtY9n"><i class="fab fa-discord"></i></a>
-              </li>
-              <li>
-                  <a target="_blank" href="mailto:gutechsoc@gmail.com"><i class="fas fa-envelope"></i></a>
-              </li>
+            <li>
+              <a target="_blank" href="https://www.facebook.com/gutechsoc"><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://twitter.com/gutechsoc"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://discord.gg/EGtY9n"><i class="fab fa-discord"></i></a>
```

Comment:
```suggestion
              <a target="_blank" href="https://discord.gg/YeemgDg"><i class="fab fa-discord"></i></a>
```

Reply:
The original link comes up as "invite invalid" for me

Reply:
Thank you for pointing this out! It was working earlier but I think at some point I must have copied an old link. I have made the suggested change now.

Reply:
Thank you for pointing this out! I think I used a link with an expiry of 24 hours (https://support.discordapp.com/hc/en-us/articles/208866998-Invites-101). I have committed the suggested change. Is the new link a 'Never' expire link with 'No limit' to the number of uses?

Reply:
Yep

Reply:
Thank you for pointing this out! I think I used a link with an expiry of 24 hours by mistake (https://support.discordapp.com/hc/en-us/articles/208866998-Invites-101). I have committed the suggested change.